### PR TITLE
libcap-ng: migrate to `python@3.11`

### PR DESCRIPTION
**Pull request by cho-m**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/libcap-ng.rb [hunk 1]
```diff
@@ -1,29 +1,32 @@
 class LibcapNg < Formula
   desc "Library for Linux that makes using posix capabilities easy"
   homepage "https://people.redhat.com/sgrubb/libcap-ng"
-  url "https://github.com/stevegrubb/libcap-ng/archive/v0.8.3.tar.gz"
-  sha256 "e542e9139961f0915ab5878427890cdc7762949fbe216bd0cb4ceedb309bb854"
-  license "LGPL-2.1-or-later"
+  url "https://people.redhat.com/sgrubb/libcap-ng/libcap-ng-0.8.3.tar.gz"
+  sha256 "bed6f6848e22bb2f83b5f764b2aef0ed393054e803a8e3a8711cb2a39e6b492d"
```
Comment (SMillerDev) [inline Formula/libcap-ng.rb:5]: Why this change? I don't see anything in the repo?

**Review by SMillerDev — CHANGES_REQUESTED** [review]

File: Formula/libcap-ng.rb (hunk 1 above)
Comment (cho-m) [reply Formula/libcap-ng.rb:5]: It's the official tarball hosted on the homepage with generated configure script as opposed to the GitHub auto-generated tarball without configure script.

You can see the information in included spec file: https://github.com/stevegrubb/libcap-ng/blob/v0.8.3/libcap-ng.spec#L7-L8
```
URL: http://people.redhat.com/sgrubb/libcap-ng
Source0: http://people.redhat.com/sgrubb/libcap-ng/%{name}-%{version}.tar.gz
```

Also can cross-reference Fedora/RHEL package given the project is maintained by them https://src.fedoraproject.org/rpms/libcap-ng/blob/rawhide/f/libcap-ng.spec

Used by:
- Alpine - https://git.alpinelinux.org/aports/tree/main/libcap-ng/APKBUILD#n6
- NixOS - https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/os-specific/linux/libcap-ng/default.nix#L8

**Review by SMillerDev — APPROVED** [review]
